js/modules/k6/grpc/client.go

```diff
@@ -116,17 +117,27 @@ func (c *Client) Load(ctxPtr *context.Context, importPaths []string, filenames .
 		return nil, errors.New("load must be called in the init context")
 	}
 
-	f, err := protoparse.ResolveFilenames(importPaths, filenames...)
-	if err != nil {
-		return nil, err
+	initEnv := common.GetInitEnv(*ctxPtr)
+	if initEnv == nil {
+		return nil, errors.New("missing init environment")
+	}
+
+	// If no import paths are specified, use the current working directory
+	if len(importPaths) == 0 {
+		importPaths = append(importPaths, initEnv.CWD.Path)
 	}
 
 	parser := protoparse.Parser{
 		ImportPaths:      importPaths,
-		InferImportPaths: len(importPaths) == 0,
+		InferImportPaths: false,
+		Accessor: protoparse.FileAccessor(func(filename string) (io.ReadCloser, error) {
+			absFilePath := initEnv.GetAbsFilePath(filename)
+			// initEnv.Logger.Infof("gRPC trying to load %s from %s...", filename, absFilePath)
+			return initEnv.FileSystems["file"].Open(absFilePath)
```

Comment:
Maybe for a follow up PR, but could we use constants / an enum for the `FileSystems` keys? It wasn't clear to me initially that this is the URL scheme.

Reply:
Yeah, we probably should refactor this together with the other `TODO`s I left...